plugin/introspection/plugin.go

```diff
@@ -0,0 +1,96 @@
+package introspection
+
+import (
+	"github.com/99designs/gqlgen/codegen"
+	"github.com/99designs/gqlgen/codegen/config"
+	"github.com/99designs/gqlgen/codegen/templates"
+	"github.com/vektah/gqlparser/ast"
+)
+
+type Config struct {
+	Directives config.StringList `yaml:"directives"`
+}
+
+type Plugin struct {
+	cfg Config
+}
+
+func New(cfg Config) *Plugin {
+	return &Plugin{cfg: cfg}
+}
+
+func (p *Plugin) Name() string {
+	return "introspection"
+}
+
+func (p *Plugin) MutateConfig(cfg *config.Config) error {
+	for _, d := range p.cfg.Directives {
+		cfg.Directives[d] = config.DirectiveConfig{SkipRuntime: true}
```

Comment:
I'm already using `@hasRole` directive, and it has runtime implementation.
I think `MutateConfig` is not necessary. 
user defines `@hide` and `@requireAuth` in schema. `skip_runtime` settings should write gqlgen.yml explicitly.

Reply:
Yes you're right, this is not necessary.